wallet-services/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -23,6 +23,7 @@ exoplatform.wallet.settings.meedsWallet=Meeds Wallet
 exoplatform.wallet.settings.useMetamask=Use Metamask
 exoplatform.wallet.settings.metamaskInstallation=Add Metamask add-on to your browser through this link
 
+exoplatform.wallet.message.followTransaction={0} link to follow your transaction
```

Comment:
add this to Wallet_en.properties please.

Reply:
done